src/Language/Haskell/GHC/ExactPrint/Transform.hs

```diff
@@ -117,7 +119,6 @@ newtype TransformT m a = TransformT { unTransformT :: RWST () [String] Int m a }
                          ,MonadReader ()
                          ,MonadWriter [String]
                          ,MonadState Int
-                         ,MonadTrans
```

Comment:
Is this requiered because of the `mtl` update on the upper classes for `MonadTrans`?

Reply:
I removed it because GHC gave a warning that it is unnecessary.  
I can re-instate it if needed.